test-requirements.txt

```diff
@@ -0,0 +1,3 @@
+-e .
+pytest
+sphinx

```

Comment:
Test requirements are already in the setup.py.

Reply:
Originally I didn't get tox working without, but now It works... remove file and commit changes.